src/Handler/Table/Import/ImportTableFromTableLib/CopyImportFromTableToTable.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\BigQuery\Handler\Table\Import\ImportTableFromTableLib;
+
+use Google\Cloud\BigQuery\BigQueryClient;
+use Google\Cloud\BigQuery\Exception\JobException;
+use Google\Cloud\Core\Exception\ServiceException;
+use Keboola\Db\ImportExport\Backend\Assert;
+use Keboola\Db\ImportExport\Backend\Bigquery\BigqueryException;
+use Keboola\Db\ImportExport\Backend\Bigquery\BigqueryImportOptions;
+use Keboola\Db\ImportExport\Backend\ImportState;
+use Keboola\Db\ImportExport\Backend\ToStageImporterInterface;
+use Keboola\Db\ImportExport\ImportOptionsInterface;
+use Keboola\Db\ImportExport\Storage;
+use Keboola\Db\ImportExport\Storage\Bigquery\Table;
+use Keboola\TableBackendUtils\Escaping\Bigquery\BigqueryQuote;
+use Keboola\TableBackendUtils\Table\Bigquery\BigqueryTableDefinition;
+use Keboola\TableBackendUtils\Table\Bigquery\BigqueryTableReflection;
+use Keboola\TableBackendUtils\Table\TableDefinitionInterface;
+
+class CopyImportFromTableToTable implements ToStageImporterInterface
```

Comment:
Místo přidávání adapteru sem to naimplementoval takto celé, stejně to chceme přesůvat mimo IE lib takže to dává tak nějak smysl.